cpp/include/cuspatial/utility/vec_2d.hpp

```diff
@@ -15,10 +15,14 @@
  */
 
 #pragma once
-#include <cuspatial/types.hpp>
-
 namespace cuspatial {
 
+#ifdef __CUDACC__
+#define CUSPATIAL_HOST_DEVICE __host__ __device__
+#else
+#define CUSPATIAL_HOST_DEVICE
+#endif
```

Comment:
I think this should be put somewhere more central. It's a bit hidden in this file.

Reply:
Moved.